pyanaconda/modules/storage/partitioning/manual/manual_module.py

```diff
@@ -73,8 +76,11 @@ def setup_kickstart(self, data):
         for request in self.requests:
             mount_data = data.MountData()
             mount_data.mount_point = request.mount_point
-            mount_data.device = request.device_spec
+
+            device = self.storage.devicetree.get_device_by_device_id(request.device_spec)
+            mount_data.device = device.path
```

Comment:
Just out of interest, could you explain why device.path was chosen to be used as device_spec, or add some context? (I guess it is available for all the devices while being unique enough ... ?) Does it have some caveats?